docs/users.rst

```diff
@@ -1092,6 +1092,13 @@ that was pushed to ``rsyslog`` will cause a build failure.
 
 This behavior requires koji integration to be enabled in the OSBS environment.
 
+For scratch builds, OSBS does not resolve a Dockerfile ``FROM`` instruction
```

Comment:
what do you mean by the 1st sentence???

Reply:
I think it's meant to capture that OSBS will not ensure there's a corresponding Koji build, i.e. not run koji_parent plugin.
I don't know how to rephrase it to make it easier to understand.